api/test/helper/routes.js

```diff
@@ -0,0 +1,71 @@
+const { buildApi } = require('../../lib')
```

Comment:
So we are going beyond just types publishing the entire package? There will be a few changes required across other repos, but like the idea of having to maintain this, but then might make sense to create it as separate repo and import from repo?

Reply:
The huge benefit of doing it here is that we can tie changes to api library to the runtime changes. That said this should be a pretty thin cover